public/js/main.js

```diff
@@ -0,0 +1,45 @@
+//Animating chat bubbles on conversation cards:
+
+window.onload = function() {
+  //find all convo cards and loop through them
+  let convoCards = document.getElementsByClassName('conversation-card')
+
+  Array.prototype.slice.call(convoCards).forEach(convo => {
+
+    //for each card, add event listener to be triggered when user scrolls down a percentage of its height
+    window.addEventListener('scroll', function(){
```

Comment:
+1 to either refactoring this to use only a single debounced scroll listener, or to use inViewport to streamline things even more.